### ISSUE TITLE
Type Mismatch Error in `multi_image_picker_view.dart`

### ISSUE THREAD

**Issue by Lukas2019** [shubham16g/multi_image_picker_view#47]
#### Description
There is a type mismatch error in the `multi_image_picker_view.dart` file. The error occurs when trying to assign a `List<ImageFile>` to a parameter that expects a `List<Never>`.

#### Error Message
```
../../../AppData/Local/Pub/Cache/hosted/pub.dev/multi_image_picker_view-1.1.0/lib/src/multi_image_picker_view.dart:114:68: Error: The argument type 'List<ImageFile>' can't be assigned to the parameter type 'List<Never>'.
- 'List' is from 'dart:core'.
- 'ImageFile' is from 'package:multi_image_picker_view/src/image_file.dart' ('../../../AppData/Local/Pub/Cache/hosted/pub.dev/multi_image_picker_view-1.1.0/lib/src/image_file.dart').
              final newList = reOrderCallback(widget.controller.images.toList());
                                                                 ^

```

#### Steps to Reproduce
1. Use the `multi_image_picker_view` package version 1.1.0.
2. Implement the `MultiImagePickerView` widget in your Flutter project.
3. Run the project and trigger the reordering functionality.

#### Expected Behavior
The `reOrderCallback` should accept a `List<ImageFile>` and reorder the images without any type mismatch errors.

#### Actual Behavior
A type mismatch error occurs because the `reOrderCallback` expects a `List<Never>` instead of a `List<ImageFile>`.

#### Possible Solution
Ensure that the `reOrderCallback` function is correctly typed to accept a `List<ImageFile>`.

#### Environment
- Dart SDK version: 3.5.4
- Flutter (Channel stable, 3.24.5, on Microsoft Windows [Version 10.0.19045.5131], locale de-DE)
- multi_image_picker_view version: 1.1.0
- Operating System: Windows 10

**Comment by jordan15** [comment]
Same issue, I'm searching a solution or by-pass to be able to run my porject

**Comment by anchao-lu** [comment]
Same issue

**Comment by Lukas2019** [comment]
Work around add `flutter reorderable grid view: 5.3.2` to your pubspec.yaml .

**Comment by shubham16g** [comment]
`flutter_reorderable_grid_view` 's latest version `5.4.0` have some breaking changes. But it is recommended not to introduce any breaking changes in same base version, in this case `5.*.*`.  Currently `multi_image_picker_view` is using it's version: `5.0.1`, but when someone adds `multi_image_picker_view` to their project, it uses the latest version of `flutter_reorderable_grid_view` with base version `5.*.*`. You can check in `pubspec.lock` file that the version of `flutter_reorderable_grid_view` is set to `5.4.0` even if `multi_image_picker_view` is set to use `5.0.1`.

**Comment by shubham16g** [comment]
Resolved in 2.0.0
